packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -454,8 +454,17 @@ export default class metadataBackup {
       throw new Error('no metadata mode found')
     }
 
-    const { retentionXoMetadata = 0, retentionPoolMetadata = 0 } =
+    let { retentionXoMetadata, retentionPoolMetadata } =
       job.settings[schedule.id] || {}
+
+    // in the old UI implementation, a retention can be null if a user delete it voluntary
```

Comment:
Has this been released?

Reply:
The UI was been released, but the old server implementation handle this case.